### PR TITLE
VS: Fix QT deploy for paths with spaces

### PR DESCRIPTION
**Pull request by al3xtjames**
This fixes VS builds when the path to the project contains spaces.

### CODE REVIEW

**Comment by JohnHolmesII** [comment]
Isn't that impossible?

**Comment by elad335** [comment]
folder names can contain spaces. I've had to reformat my computer in the past so my main username won't contain them and building would work.

**Comment by JohnHolmesII** [comment]
But not the Qt folders

**Comment by JohnHolmesII** [comment]
Ah, I see. You'll have to forgive me my mind replaced CMAKE with QTDIR magically. I've been up too long today 😄 

Carry on then

File: rpcs3/rpcs3.vcxproj [hunk 1]
```diff
@@ -166,7 +166,7 @@
     </ResourceCompile>
     <PostBuildEvent>
       <Command>
-        $(QTDIR)\bin\windeployqt --no-angle --no-opengl-sw --no-svg --no-translations --no-quick --plugindir $(TargetDir)\qt\plugins $(TargetPath)
```
Comment (henriquenj) [inline rpcs3/rpcs3.vcxproj]: Does `$(TargetDir)` always contain a trailing backslash?

Comment (al3xtjames) [reply rpcs3/rpcs3.vcxproj]: It should: https://msdn.microsoft.com/en-us/library/c02as0cs.aspx

Comment (henriquenj) [reply rpcs3/rpcs3.vcxproj]: I understand, thanks for the reference. The change seems correct.

**Pull request merged**